scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError, NativePaymentProcessingError
+            HTTP, GraphQL, UserError, NativePaymentProcessingError, CheckoutError
```

Comment:
Is `CheckoutError` too broad? One might wonder if they need to handle a `CheckoutError` on all checkouts.  Currently it is only thrown when verification fails, perhaps `WebCheckoutVerificationError`